packages/contracts-bedrock/src/cannon/libraries/MIPS64Instructions.sol

```diff
@@ -82,7 +101,7 @@ library MIPS64Instructions {
             if (_args.opcode == 2 || _args.opcode == 3) {
                 // Take top 4 bits of the next PC (its 256 MB region), and concatenate with the 26-bit offset
                 uint64 target = (_args.cpu.nextPC & signExtend(0xF0000000, 32)) | uint64((_args.insn & 0x03FFFFFF) << 2);
-                handleJump(_args.cpu, _args.registers, _args.opcode == 2 ? 0 : REG_RA, target);
+                handleJump(_args, _args.opcode == 2 ? 0 : REG_RA, target);
```

Comment:
Just curious why we updated `handleJump` - doesn't look like the logic changed here.

Reply:
it's to get around out of stack issues. Collapsing both `cpu` and `registers` arguments into one eliminates compiler errors.